libcontainer/rootfs_linux.go

```diff
@@ -572,6 +572,13 @@ func pivotRoot(rootfs, pivotBaseDir string) error {
 	if err != nil {
 		return fmt.Errorf("can't create pivot_root dir %s, error %v", pivotDir, err)
 	}
+	defer func() error {
+		err := os.Remove(pivotDir)
```

Comment:
@mikebrow 
I would like to return if there is any error return from os.Remove, i.e the reason I'm checking the return value. Also, earlier it was return os.Remove. I'm committing with named value shortly